src/dsc.h

```diff
@@ -521,6 +521,13 @@ class IdList {
         ++n;
     }
 
+    T *FindByHandleV(uint32_t v) {
```

Comment:
This doesn't really make things better, I want to get rid of this and looking up entities in the sketch by just returning the full struct of Entities / Constraints from the helper methods, then the lookup can be completely removed.